app/controllers/concerns/breadcrumbed.rb

```diff
@@ -0,0 +1,22 @@
+module Breadcrumbed
+  extend ActiveSupport::Concern
+
+  def prepare_default_activity_trail(activity)
+    if activity.historic?
+      add_breadcrumb "Historic activities", historic_organisation_activities_path(activity.organisation)
```

Comment:
Could we use translations for the "Historic activities" and "Current activities" breadcrumb titles? There's a lot of repetition in the other controllers

Reply:
Thank you!